frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -484,6 +484,266 @@ describe('NewRunParametersV2', () => {
     screen.getByDisplayValue('"A":1,"B":2');
   });
 
+  it('test Start button is enabled for valid default integer input', () => {
```

Comment:
Will `setIsValidInputSpy` be called with `false` when there is no default input? Can we add a test for it?